### DISCUSSION TITLE
Wazuh as a Backend for Sigma

### DISCUSSION THREAD

**Discussion by Number-39** [SigmaHQ/pySigma-plugin-directory#52] [Ask for Backends]
Is there any possibility that Wazuh SIEM becomes a supported backend for Sigma? If not, is there any way to convert Sigma rules to Wazuh language?

**Comment by thomaspatzke** [comment]
Yes, someone has to implement it, preferred someone who has access to a Wazuh instance to being able to test.

Moving to Sigma plugin directory discussions.
